src/main/java/com/revature/entities/Question.java

```diff
@@ -50,11 +48,11 @@ public void setId(int id) {
 		this.id = id;
 	}
 
-	public int getAcceptedId() {
+	public Integer getAcceptedId() {
```

Comment:
Initially I had made this a wrapper class so that it could be set to a null value. I don't remember if we found another way to make this work, if so we can switch it back to int.